doc/_static/css/custom.css

```diff
@@ -9,11 +9,11 @@ p {
 .wy-nav-content {
     max-width: 950px;
 }
-
+/**
 .wy-side-nav-search>a, .wy-side-nav-search .wy-dropdown>a {
    content: url(../../_static/img/nest_logo.png);
 }
-
+**/
```

Comment:
I assume `/** ... **/` comments out lines 13-15. If the code is not used any more, please remove it.

Reply:
Done